src/Orleans.Core.Abstractions/Placement/ResourceOptimizedPlacement.cs

```diff
@@ -0,0 +1,19 @@
+namespace Orleans.Runtime;
+
+/// <summary>
+/// A placement strategy which attempts to optimize resource distribution across the cluster.
+/// </summary>
+/// <remarks>
+/// <para>It assigns weights to runtime statistics to prioritize different resources and calculates a normalized score for each silo.
+/// The silo with the lowest score is chosen for placing the activation. Normalization ensures that each property contributes proportionally
```

Comment:
"the silo with the lowest score is chosen" << this is not quite accurate, since you're using an adaptive PowOfK

Reply:
true, I should make that clear